resources/views/posts/add.blade.php

```diff
@@ -7,18 +7,29 @@
                 <div class="card">
                     <div id = "header" class="card-header">Post</div>
                     <div class="card-body">
+                        <form id = "post_form">
                             <div class="row mb-3">
                                 <label for="title" class="col-md-4 col-form-label text-md-end">Post Title</label>
 
                                 <div id = "titlediv" class="col-md-6">
                                     <input id="title" type="title" class="form-control @error('title') is-invalid @enderror" name="title" required autocomplete="title" autofocus>
+                                    @error('title')
+                                        <span class="invalid-feedback" role="alert">
+                                            <strong>{{ $message }}</strong>
+                                        </span>
+                                    @enderror
```

Comment:
@error directives won't work when sending api requests. To display the validations via api, you will have to use frontend scripting. In your case, you may use jquery/javascript